doc/planning/new_powertoy_ideas.md

```diff
@@ -0,0 +1,305 @@
+* [x] #1240 isn't a new PowerToy. More a new Item for Settings.
+   * totally can see this being a PowerToy.
+* [x] #46 is already in windows 10
+* [x] I think #66 is the same as #43
+* [x] #85 is already something suggested in #73 (Duplicate: #4189 , #4663)
+   * crutkas: #73vs#85, they are different asks
+* [x] #1331 is duplicate of #472
+* [x] #2615 is too dangerous
+* [x] #2899 and #5721 are the same as #115
+* [x] #2954 is duplicate of #278
+* [x] #3064 is in #1196
+* [x] #3096 is also in #825 (Duplicate: #5083 , #5657)
+* [x] #3217 is more a bug then a PowerToy idea
+* [x] #3531 duplicate of #26
+* [x] #3570 is duplicate of #10
+* [x] #3982 is duplicate of #1642
+* [x] #4408 also is in #1951
+* [x] #5466 is in #3442
+* [x] #5499 duplicate of #3583
+* [x] #6244 duplicate of #5074
+
+# Difficulty (My opinion. I'm not a expert)
+## Very Very Difficult or impossible
+* #7
+* #18
+* #47
+* #59
+* #73
+* #81
+* #84
+* #107 (I mean how should that work)
+* #110
+* #112
+* #124
+* #130
+* #134
+* #148
+* #149
+* #155
+* #264
+* #470
+* #493
+* #503
+* #717
+* #1052 (Issue with many other linked)
+* #1143 (Not possible as suggested) 
+* #1305
+* #1354
+* #1392 (Can be added to #5476)
+* #1488
+* #1489
+* #1609
+* #1839
+* #1846
+* #1908
+* #1988
+* #2067
+* #2083
+* #2084 (How should that work?)
+* #2652
+* #2769
+* #3526 (I think not possible, because every app has its own preview)
+* #3885
+* #3913
+* #3954 (Not possible, if it isn't from Windows itsself)
+* #4464
+* #4567
+* #5230 (Not good, because the Control Panel will not long exist)
+* #5476
+* #5580
+* #5807
+* #5836
+* #5901
+* #6015
+* #6102
+* #6141 
+* #6325
+* #6157
+*
+
+## Middle
+* #10
+* #13
+* #26
+* #43
+* #63
+* #82 (But it is in another way implemented in Windows 10)
+* #86
+* #91
+* #96
+* #114
+* #119
+* #123
+* #128
+* #143
+* #146
+* #233
+* #322
+* #360
+* #419
+* #426
+* #454
+* #457
+* #472
+* #753
+* #774
+* #799
+* #825 (can be added to #1354) (see also #3096)
+* #834
+* #907
+* #973
+* #1011
+* #1084
+* #1352
+* #1870
+* #1984
+* #2555
+* #2668
+* #2927
+* #3143 (Can be added to #5476)
+* #3091
+* #3334
+* #3388
+* #4167
+* #4246
+* #4371
+* #4999
+* #5074
+* #5269
+* #5871
+* #6357
+*
+
+## Easy
+* #14
+* #19
+* #75
+* #67
+* #139
+* #145
+* #278
+* #828
+* #892
+* #977
+* #984
+* #995
+* #1064
+* #1465
+* #1642
+* #3353
+* #3442
+* #3583
+* #3754
+* #3765
+* #4859
+* #5472
+* #863
+*
+
+## I don't know the difficulty or I don't understand the idea
+* #11
+* #16
+* #22
+* #58
+* #61
+* #65
+* #74
+* #77
+* #89
+* #105
+* #121
+* #142
+* #169
+* #353
+* #445
+* #557
+* #586
+* #988
+* #1056
+* #1291
+* #1527
+* #1535
+* #1679
+* #1919
+* #2343
+* #2682
+* #2841
+* #3098
+* #3458
+* #3768 (maybe a duplicate of another issue)
+* #4162
+* #5088
+* #5569
+* #5859
+* #6190
+* #6273
+* 
+
+# Implement in other PowerToy
+## To FanzyZones
+* #4
```

Comment:
It's "FancyZones"

Reply:
oh i'm so dumb. Oh your now a collaborator. Congratulations!

Reply:
Yep. Thanks